test/testdrive/basic.td

```diff
@@ -526,6 +526,27 @@ a b
 1 2
 2 3
 
+$ kafka-ingest format=raw topic=test-bytes timestamp=1
+test-string
+
+> CREATE SOURCE test_bytes
```

Comment:
We support materialized sources now, so you can just create a materialized source here and avoiding creating the extra view!